keystore/keyloader/kms/kms_loader.go

```diff
@@ -0,0 +1,106 @@
+package kms
+
+import (
+	"crypto/subtle"
+
+	"github.com/cossacklabs/acra/keystore"
+	keystoreCE "github.com/cossacklabs/acra/keystore"
+	"github.com/cossacklabs/acra/keystore/kms"
+	"github.com/cossacklabs/acra/keystore/kms/aws"
+	keystoreV2CE "github.com/cossacklabs/acra/keystore/v2/keystore"
+	log "github.com/sirupsen/logrus"
+)
+
+// Loader is implementation of MasterKeyLoader for kms
+type Loader struct {
+	keyID     kms.KeyIdentifier
+	encryptor kms.Encryptor
+}
+
+// NewLoader create new kms MasterKeyLoader
+func NewLoader(credentialPath, keyIdentifierURI string) (*Loader, error) {
+	keyID, err := kms.NewKeyIdentifierFromURI(keyIdentifierURI)
+	if err != nil {
+		return nil, err
+	}
+
+	var encryptor kms.Encryptor
+	switch keyID.Prefix() {
```

Comment:
hm, it push me to think about registries. what about to use here registries approach as we did it for CryptoHandlers? Turn implementation on with build tag and register it at start? for example:
```
type EncryptorFactoryMethod interface {
  New(credentialPath string) (kms.Encryptor, error)
}
// or type EncryptorFactoryMethod func(credentialPath string) (kms.Encryptor, error)
var lock = sync.Lock{}
var encryptors map[string]EncryptorFactoryMethod
RegisterEncryptor(encryptor EncryptorFactoryMethod){
  lock.Lock()
  encryptors[encryptor.ID()] = encryptor
  lock.Unlock()
}
```

Then we can use :
```
func NewLoader(credentialPath, keyIdentifierURI string) (*Loader, error) {
	keyID, err := kms.NewKeyIdentifierFromURI(keyIdentifierURI)
	if err != nil {
		return nil, err
	}
        encryptorFactory, ok := encryptors[keyID]
        if !ok {
            log.Errorln("Unknown key ID")
        }
        encryptor, err := encryptorFactory.New(credentialPath) // or encryptorFactory(credentialPath)
}
```

Then our AWS implementation should have:
```
func init(){
  kms.RegisterEncryptor(someFactoryFunction)
}
```
With such approach we can extend with enterprise KMS support.


Additionally, we can make default loader that do nothing with master key (current approach where env variable store result master key. this implementation will return master key as is in Encrypt/Decrypt methods) and registered by default and +1 with AWS. Then current loader will be consistent with new approach